cli/tools/repl.rs

```diff
@@ -444,7 +447,7 @@ impl ReplSession {
   }
 
   pub async fn run_event_loop(&mut self) -> Result<(), AnyError> {
-    self.worker.run_event_loop(false).await
+    self.worker.run_event_loop(true).await
```

Comment:
`wait_for_inspector` is the boolean here. When it's `false` the inspector doesn't work without hitting enter a few times in the repl, but once I set it to `true` then everything started working properly.